optuna_dashboard/ts/apiClient.ts

```diff
@@ -137,6 +138,15 @@ export enum CompareStudiesPlotType {
   EDF = "edf",
 }
 
+export type TrialFilterQueryLastResponse = {
+  func_str: string
+  error_message: string
+}
+
+export type TrialFilterQueryResponse = {
+  trial_filtering_func_str: string
+}
```

Comment:
How about defining a `TrialFilterQueryRequest` type to clearly distinguish the request and response structures?

```suggestion
export type TrialFilterQueryRequest = {
  user_query: string
  last_response?: {
    func_str: string
    error_message: string
  }
}

export type TrialFilterQueryResponse = {
  trial_filtering_func_str: string
}
```

Reply:
Thank you. I applied your suggestion.